pkgs/applications/office/scribus/unstable.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchsvn, makeWrapper, pkgconfig, cmake, qtbase, cairo, pixman,
+boost, cups, fontconfig, freetype, hunspell, libjpeg, libtiff, libxml2, lcms2,
+podofo, poppler, poppler_data, python2, harfbuzz, qtimageformats, qttools }:
+
+let
+  pythonEnv = python2.withPackages(ps: [ps.tkinter ps.pillow]);
+  revision = "22716";
+in 
+stdenv.mkDerivation rec {
+  name = "scribus-${version}";
```

Comment:
The package name should be: `scribus-unstable-2018-10-13`. You could also add a suffix is you want.

Reply:
I see. should I also change the attribute name or can it stay at `scribusUnstable`?

Reply:
I pushed the change of the package name (also bumped to newest rev) and kept the old attribute name,